.github/renovate.json5

```
@@ -40,6 +40,10 @@
       "groupName": "python-nonmajor",
       "matchLanguages": ["python"],
       "matchUpdateTypes": ["minor", "patch"]
+    },
+    {
+      "groupName": "kokoro-deps",
+      "matchFileNames": [".kokoro"],
```

It looks like it uses minimatch -- does it need a globstar?
```suggestion
      "matchFileNames": [".kokoro/**"],
```
https://docs.renovatebot.com/configuration-options/#matchfilenames

---

Thanks for catching this 🙂